experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -191,3 +191,35 @@ def __init__(self, *args, **kwargs):
 
     def get_changelog_message(self):
         return f"{self.request.user} updated metrics"
+
+
+class SubscribeForm(NimbusChangeLogFormMixin, forms.ModelForm):
+    class Meta:
+        model = NimbusExperiment
+        fields = []
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop("user", None)
+        super().__init__(*args, **kwargs)
+
+    def save(self, commit=True):
+        self.instance.subscribers.add(self.user)
+        if commit:
+            self.instance.save()
```

Comment:
Do you need this here?  If you do I'd do it as

```py
experiment = super().save(commit=commit)
```

But if the only thing this form is doing is adding the subscriber, this shouldn't be necessary?